src/CreateTableQuery.ts

```diff
@@ -18,7 +18,7 @@ export class CreateTableQuery {
 
     private orderByPart: Array<string> = [];
 
-    constructor(ch: ClickHouse, logger: Logger | null) {
+    constructor(ch: ClickHouseLike | null, logger: LoggerLike | null) {
```

Comment:
Is there a specific reason you've made `ch` argument optional? It must be required, otherwise it's class itself is pointless.